src/javaRosa.js

```diff
@@ -992,29 +987,20 @@ define([
             $input.val(val);
         };
 
-        widget.setPlaceholder = function (val) {
-            $input.attr("placeholder", val);
-        };
-
         widget.getValue = function () {
             return $input.val();
         };
 
-        widget.getPlaceholder = function () {
-            return $input.attr('placeholder');
-        };
-
         widget.getDefaultValue = function () {
             return null;
         };
 
-        if (widget.hasNodeIdPlaceholder && widget.isDefaultLang) {
+        if (widget.hasNodeIdAsDefault && widget.isDefaultLang) {
             widget.mug.on('property-changed', function (e) {
                 if (e.property === "nodeID") {
-                    widget.setPlaceholder(e.val);
                     if (widget.getItextValue() === e.previous || !widget.getValue()) {
```

Comment:
I think `|| !widget.getValue()` should be removed.

- Create text question
- Clear default language label
- Change question ID

Expected: label remains empty
Observed: label changes to question ID